apps/torus-allocator/src/hooks/use-weekly-usd.ts

```diff
@@ -47,64 +34,66 @@ export function useWeeklyUsdCalculation(
   } = extAPI.computedAgentWeight.byAgentKey.useQuery({ agentKey: agent.key });
 
   // Loads all queries at once, and if any of them are wrong, the whole result is wrong
-  const isLoading =
-    isEmissionLoading ||
-    isIncentivesLoading ||
-    isTorusPriceLoading ||
-    isComputedWeightLoading;
-
-  const isError =
-    isEmissionError ||
-    isIncentivesError ||
-    isTorusPriceError ||
-    isComputedWeightError;
+  const isLoading = isTorusPriceLoading || isComputedWeightLoading;
+  const isError = isTorusPriceError || isComputedWeightError;
 
   // Calculate tokens per week
   const tokensPerWeek = useMemo(() => {
-    if (
-      !emission ||
-      !incentivesRatio ||
-      !computedWeightedAgents?.computedWeight
-    )
-      return 0;
+    if (!computedWeightedAgents?.computedWeight) return 0;
 
     // Blocks per week calculation
-    const BLOCKS_PER_WEEK = ONE_WEEK / BLOCK_TIME_SECONDS;
+    const BLOCKS_PER_WEEK =
+      CONSTANTS.TIME.ONE_WEEK / CONSTANTS.TIME.BLOCK_TIME_SECONDS;
+
+    // Weight Factor is the penalty factor
     const weightPenaltyFactor = agent.weightFactor ?? 1; // Default to 1 if not available
 
     // Calculate weekly emission in NANOs and convert to tokens
-    const weeklyEmission = Number(fromNano(emission)) * BLOCKS_PER_WEEK;
+    const weeklyEmission = CONSTANTS.EMISSION.BLOCK_EMISSION * BLOCKS_PER_WEEK;
```

Comment:
This is tecnically constant right now, but will be dynamic in the future and we should account for that.